lightstep/resource_metric_dashboard.go

```diff
@@ -18,6 +18,10 @@ type ChartSchemaType int
 const (
 	MetricChartSchema ChartSchemaType = iota
 	UnifiedChartSchema
+
+	// if the incoming HCL contains loose charts, we will put them in an implicit group
+	// this group should have a unique name, so we can distinguish it from an _explicit_ implicit group with no title
+	legacyImplicitGroupName = "legacy implicit group"
```

Comment:
yes, someone might explicitly name their implicit group`"legacy implicit group"`, which would break this and result in a spurious diff. this seems exceedingly unlikely. why would anyone name their implicit group anyway.

Reply:
By definition, if an implicit group is named, it is an explicit group